start_cerberus.py

```diff
@@ -492,6 +500,10 @@ def main(cfg):
             record_time(time_tracker)
             pool.close()
             pool.join()
+            if cerberus_publish_status:
+                final_status = get_cerberus_status()
+                logging.info("Final number status" + str(final_status))
+                sys.exit(final_status)
```

Comment:
Think the cerberus exit status should only reflect the code exit status instead of cluster health status given that users will be relying on the go/no-go signal vs cerberus exit code. Thoughts?

Reply:
Combining Cerberus code exit code with cluster health exit code will make it confusing as we will never know the reason behind as whether there was an exception or the cluster is unhealthy IMHO.

I think we can solve the use case by dumping a json in the run dir at the end summarizing the run, something like:

{
   iterations: 5,
   code_exit_status: 0,
   cluster_health: 0
}

Thoughts?

Reply:
Think that sounds good! Will add that to this PR

Reply:
@chaitanyaenr are you able to have a look at this PR, I added in a write to file like you gave as an example